src/status_im2/contexts/communities/actions/community_options/view.cljs

```diff
@@ -45,12 +48,24 @@
    :on-press            #(hide-sheet-and-dispatch [:chat.ui/mark-all-read-in-community-pressed id])})
 
 (defn mute-community
-  [_ muted?]
+  [id muted? muted-till]
   {:icon                (if muted? :i/muted :i/activity-center)
    :accessibility-label (if muted? :unmute-community :mute-community)
    :label               (i18n/label (if muted? :t/unmute-community :t/mute-community))
-   :sub-label           (when muted? (str "muted for 15 minutes"))
-   :right-icon          :i/chevron-right})
+   :sub-label           (when (and muted? (some? muted-till))
+                          (str (i18n/label :t/muted-until) " " (datetime/format-mute-till muted-till)))
```

Comment:
Hey @jo-mut - I think this i18n string should be using a param instead.
e.g `(i18n/label :t/muted-until {:time (datetime/format-mute-till muted-till)})`

you would have to add the parameter to the translation string in en.json too ofc 👍